Add a note about .terraformignore to CLI workflow (#960)
hashicorp/terraform-website: content/source/docs/cloud/run/cli.html.md

If you use a combined repository and [specify a working directory on workspaces](../workspaces/settings.html#terraform-working-directory), you can run Terraform from either the real working directory or from the root of the combined configuration directory. In both cases, Terraform will upload the entire combined configuration directory.

> ## Excluding files from upload

Reviewer: ```suggestion
## Excluding Files from Upload
```

Title-case for headers.

Reviewer: (Sidenote: while I was second-guessing myself on that "from," I found this, which IMHO is excellent. https://titlecaseconverter.com/words-to-capitalize/?style=AP,CMOS,MLA,NYT,WP#from)

> When using in the CLI-driven run workflow, an archive of your configuration directory is

Reviewer: ```suggestion
CLI-driven runs upload an archive of your configuration directory
```

Author: Thanks, I was trying to follow/see if "run workflow" was some sort of terminology we'd decided to stick to, but I find it confusing. Grateful to simplify :)

> uploaded to Terraform Cloud. There may be files you would like to exclude from upload,

Reviewer: ```suggestion
to Terraform Cloud. If the directory contains files you want to exclude from upload,
```

> and you can do so by defining a [`.terraformignore` file in your configuration directory][remote].

Reviewer: ```suggestion
you can do so by defining a [`.terraformignore` file in your configuration directory][remote].
```
